src/OrbitGl/CallTreeView.cpp

```diff
@@ -208,13 +213,13 @@ static void AddUnwindErrorToTopDownThread(
 
 std::unique_ptr<CallTreeView> CallTreeView::CreateTopDownViewFromPostProcessedSamplingData(
     const PostProcessedSamplingData& post_processed_sampling_data,
-    const ModuleManager& module_manager, const CaptureData& capture_data) {
+    const ModuleManager* module_manager, const CaptureData* capture_data) {
   ORBIT_SCOPE_FUNCTION;
   ORBIT_SCOPED_TIMED_LOG("CreateTopDownViewFromPostProcessedSamplingData");
 
-  auto top_down_view = std::make_unique<CallTreeView>();
-  const std::string& process_name = capture_data.process_name();
-  const absl::flat_hash_map<uint32_t, std::string>& thread_names = capture_data.thread_names();
+  std::unique_ptr<CallTreeRoot> top_down_view_root = std::make_unique<CallTreeRoot>();
```

Comment:
Would clang-tidy suggest
```suggestion
  auto top_down_view_root = std::make_unique<CallTreeRoot>();
```
?

Reply:
Same in `CreateBottomUpViewFromPostProcessedSamplingData`.

Reply:
Done